app/src/main/java/com/example/mycalculator/utils/ConstantsUtils.java

```diff
@@ -0,0 +1,8 @@
+package com.example.mycalculator.utils;
+
+public class ConstantsUtils {
```

Comment:
you can unify StringUtils and ConstantsUtils class

Reply:
Done